zk/syncer/l1_syncer.go

```diff
@@ -302,9 +302,9 @@ func (s *L1Syncer) getLatestL1Block() (uint64, error) {
 }
 
 func (s *L1Syncer) queryBlocks() error {
-	startBlock := s.lastCheckedL1Block.Load()
+	startBlock := s.lastCheckedL1Block.Load() + 1
```

Comment:
why +1 here?

Reply:
It occasionally receives duplicate log events. `lastCheckedL1Block` indicates that it has already been checked in the previous cycle. Therefore, it should not be checked again in the new cycly and +1 here.

Reply:
can you add a comment for that?

Reply:
Done